--- a/Source/AkaneCapture/AkaneCapture.csproj     2019-10-27 15:59:26.544860700 +0900
+++ b/Source/AkaneCapture/AkaneCapture.csproj     2019-10-27 15:59:11.577799200 +0900
@@ -7,6 +7,8 @@
     <Platforms>x64</Platforms>
     <Platform>x64</Platform>
     <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
+    <PublishSingleFile>true</PublishSingleFile>
+    <PublishTrimmed>true</PublishTrimmed>
   </PropertyGroup>

   <ItemGroup>
